middlewares/validators/user.js

```diff
@@ -54,15 +54,15 @@ const validateJoinData = async (req, res, next) => {
     .strict()
     .keys({
       userId: joi.string().optional(),
-      firstName: joi.string().required(),
-      lastName: joi.string().required(),
-      college: joi.string().min(5).required(),
+      firstName: joi.string().min(1).required(),
+      lastName: joi.string().min(1).required(),
+      college: joi.string().min(1).required(),
       skills: joi.string().min(5).required(),
```

Comment:
What if someone has only one skill like `c` only or no skill, and is joining for skills development

Reply:
They can also mention their personal skills and use sentences like "I am good at x y z"

Reply:
it was mandatory to use 5 words in skills